client/js/ui.js

```diff
@@ -1590,24 +1601,24 @@ function itemIcon(key) {
   pic = [0, 8, 24];
 
   let user = PlayerWho[key];
-  if(key in PlayerImages) {
-    if(PlayerImages[key].naturalWidth != 16 || PlayerImages[key].naturalHeight != 16) {
+  if (key in PlayerImages) {
+    if (PlayerImages[key].naturalWidth != 16 || PlayerImages[key].naturalHeight != 16) {
       img.style.width = "32px";
       img.style.height = "32px";
     }
 
     src = PlayerImages[key].src;
   }
 
-  if(item.pic != null)
+  if (item != undefined && item.pic != null)
```

Comment:
I think this could be:
```suggestion
  if (item?.pic)
```

Reply:
Yeah, it can. I'm never sure, coming into a project, what the style is going to be, but I guess optional chaining has been around long enough that it's safe (also, now that I look, it's used elsewhere in this same file).